blog_system/settings.py

```diff
@@ -0,0 +1,140 @@
+"""
+Django settings for blog_system project.
+
+Generated by 'django-admin startproject' using Django 5.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = (
+    "django-insecure-4k0(4a&fy72zq-i&z!@)2ofy$3%c(2@j++u_m2!0q2qln17vcq"
```

Comment:
The `SECRET_KEY` should not be hardcoded in the settings file for production environments. Consider using environment variables to manage sensitive information securely.